jerry-core/parser/js/js-lexer.c

```diff
@@ -1135,6 +1135,9 @@ lexer_parse_string (parser_context_t *context_p, /**< context */
       length += 2 * 3;
       has_escape = true;
       source_p += 4;
+#if ENABLED (JERRY_ES2015)
+      raw_length_dec -= 2;
```

Comment:
I don't think this can be < 0, so why change it to int32?

Reply:
I can be. in the related testcase  `raw_length_dec` - 2.